hw/ip/csrng/csrng.core

```diff
@@ -66,6 +66,10 @@ targets:
     parameters:
       - SYNTHESIS=true
     tools:
+      ascentlint:
+        ascentlint_options:
+          - "-wait_license"
+          - "-stop_on_error"
```

Comment:
this is not needed anymore (we found a way to put this into a common config).

Reply:
Yes, was un-intended, thought it was gone.